.chloggen/users_sourabhjain_otelcosmosmetrics.yaml

```
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: db
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Added new common `db.response.returned_rows` database metric and several operation level metrics for Azure Cosmos DB."
```

sorry @sourabh1007 for the confusion, the attribute name (when we introduce it) would be `db.response.returned_rows`, but the metric name should be `db.client.response.returned_rows`

this is similar to the situation in http, e.g.
* metric name: `http.client.response.body.size`
* attribute name `http.response.body.size`


```suggestion
note: "Added new common `db.client.response.returned_rows` database metric and several operation level metrics for Azure Cosmos DB."
```